### DISCUSSION TITLE
Prone positioning

### DISCUSSION THREAD

**Discussion by thaiswalter10** [MIT-LCP/mimic-code#1616] [MIMIC-IV]
For intubated patients, does the MIMIC IV database provide the information on whether patients are ventilated on the supine or on the prone position ? 
Thank you very much in advance !

**Comment by alistairewj** [comment]
Proning wasn't done in the hospital until COVID, so as of MIMIC-IV v2.2 you're unlikely to any patients who are proned (the last discharge was in 2019). This will change in future versions of MIMIC when COVID patients are added.

**Reply by thaiswalter10** [reply]
Starting from MIMIC v3.0 you incorporated patients from the COVID waves. Would you say that proning sessions and duration were consistently recorded in this updated version ? i.e. that when patients were turned prone, it was systematically recorded in the EHR with the precise starting and ending date and hour ? 

Thank you very much,
